Methane_MarkDown.Rmd

```diff
@@ -21,53 +21,58 @@ Our first step is to create a core. Hector will use this to pull data from exist
 ```{r  Hector core}
 library(hector)
 library(ggplot2)
+library(dplyr)
 ini_file <- system.file("input/hector_ssp245.ini", package = "hector")
 core <- newcore(ini_file)
 run(core)
 ```
 
 ## Methane Emmisions Data and Graph
 
-After creating the core we then tell hector to provide data for `EMISSIONS_CH4()` between the years of 1795 and 2100. Hector has several options that relate to methane- these include: emissions, concentration, postindustrial, natural, soil lifetime and stratosphere lifetime.
+After creating and running the core we query it for methane-related outputs
+between the years of 1795 and 2100. Hector has several options that relate to methane, including emissions, concentration, postindustrial, natural, soil lifetime and stratosphere lifetime.
 
 ```{r Emissions data frame}
-results<- fetchvars(core, 1795:2100, EMISSIONS_CH4())
+results<- fetchvars(core, 1795:2100, c(EMISSIONS_CH4(),
```

Comment:
Just curious, why 1795?

Reply:
I picked it randomly thinking it was the beginning of the data set- now that you mention it I'll change it to the actual beginning.